### PR TITLE
Support prompt login

### PR DESCRIPTION
**Pull request by jesseyay**
Hi team, thanks for the great library!

## Description of the Change
Adds support for `prompt=login` to comply with https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [x] unit-test added
- [x] documentation updated
- [x] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: docs/oidc.rst [hunk 1]
```diff
@@ -359,6 +359,15 @@ token, so you will probably want to re-use that::
...
             return claims
 
+Customizing the login flow
+==========================
+
+Clients can request that the user logs in each time a request to the
+``/authorize`` endpoint is made during the OIDC Authorization Code Flow by
+adding the ``prompt=login`` query parameter and value. Only ``login`` is
+currently supported. See
+`OIDC Docs <https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest>`_
```
Comment (n2ygk) [inline docs/oidc.rst:369]:
```suggestion
OIDC's `3.1.2.1 Authentication Request <https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest>`_
```

**Review by n2ygk — CHANGES_REQUESTED** [review]
Hey thanks! Please see a couple of documentation suggestions.

**Comment by jesseyay** [comment]
> Hey thanks! Please see a couple of documentation suggestions.

Yep makes sense, thanks for the review :)

**Comment by n2ygk** [comment]
darn it, I tried to rebase to fix the CHANGELOG conflict...

**Review by n2ygk — APPROVED** [review]

**Pull request merged**